### PR TITLE
docs(replace): describe process.env.NODE_ENV case

### PR DESCRIPTION
**Pull request by TrySound**
## Rollup Plugin Name: `replace`

This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [x] documentation
- [ ] other

Are tests included?

- [ ] yes (_bugfixes and features will not be merged without tests_)
- [x] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

If yes, then include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking. 

List any relevant issue numbers:

### Description

Replacing process.env.NODE_ENV is the most popular case for this plugin. We should have it in readme.
See here https://github.com/ai/nanoid/issues/236

### CODE REVIEW

File: packages/replace/README.md [hunk 1]
```diff
@@ -52,6 +52,25 @@ The configuration above will replace every instance of `__buildEnv__` with `'pro
...
+
+export default {
+  input: 'src/index.js',
+  output: {
+    dir: 'output',
+    format: 'cjs'
+  },
+  plugins: [
+    replace({
+      'process.env.NODE_ENV': JSON.stringify('production') // JSON.stringify('development')
```
Comment (ai) [inline packages/replace/README.md:68]:
```suggestion
      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
```

Will it be more universal way?

Comment (TrySound) [reply packages/replace/README.md:68]: I usually use config array and specify all values explicitly. NODE_ENV may raise additional question how to pass it to rollup.

**Review by Andarist — APPROVED** [review]

File: packages/replace/README.md (hunk 1 above)
Comment (shellscape) [reply packages/replace/README.md]: I took the best of both your comments to add a comment to the code explaining alternative values. hopefully that will be clear for people who take both approaches.

Comment (TrySound) [reply packages/replace/README.md]: Good

**Pull request merged**